source/Features.rst

```diff
@@ -105,7 +99,7 @@ You can filter by O.U, Filter by role(Admin/Researcher/Principal Investigator),
 .. image:: images/filter.png
 .. image:: images/sort.png
 
-We have an active filter: enable and disable option. You can toggle between active or all users.
+The user can see an active filter with enable and disable option. You can toggle between active or all users.
```

Comment:
Use "options" (plural).
Also replace "You can toggle between active or all users." with "You can toggle the view between active or all users."